relay-general/src/store/transactions/processor.rs

```diff
@@ -1808,14 +1806,7 @@ mod tests {
 
         process_value(
             &mut event,
-            &mut TransactionsProcessor::new(
-                TransactionNameConfig {
-                    rules: &[],
-                    ready: true,
-                },
-                false,
-                None,
-            ),
+            &mut TransactionsProcessor::new(TransactionNameConfig { rules: &[] }, false, None),
```

Comment:
```suggestion
            &mut TransactionsProcessor::default(),
```
Any reason to not do this?